src/os/bluestore/Allocator.cc

```diff
@@ -6,6 +6,7 @@
 #include "BitmapAllocator.h"
 #include "AvlAllocator.h"
 #include "HybridAllocator.h"
+#include "ZonedAllocator.h"
```

Comment:
could guard this include with `#ifdef HAVE_LIBZBC`

Reply:
Done.